defaults/main.yml

```diff
@@ -24,24 +29,27 @@ postgres_labels:
   kubernetes.io/cluster-service: "true"
 
 postgres_volumes:
+    # PGDATA volume
     - name: varlibpostgres
-      hostPath: 
-        path: /var/lib/postgres/data
+      hostPath:
+        path: /var/lib/postgresql/data
```

Comment:
host path should be specific to the pod, it will collide with multiple kubernetes domains. 
I propose /containers/{{domain-name}}/rc-{{rc-name}}/rootfs//var/lib/postgresql/data